index.html

```diff
@@ -1565,45 +1565,40 @@ <h3>Service properties</h3>
     </section>
 
     <section>
-      <h2>DID Subject</h2>
+      <h2>Identification</h2>
 
       <p>
-  The <a>DID subject</a> is the entity that the <a>DID document</a> is about.
-  That is, it is the entity identified by the <a>DID</a> and described by the
-  <a>DID document</a>.
+It is useful to be able to identify the <a>DID subject</a>, the <a>DID
+controller</a>, and any alternate identifiers that the <a>DID subject</a>
+might also be known as. The following section elaborates on the mechanisms for
+doing so in the <a>DID document</a> data model.
       </p>
 
       <section>
-        <h3>
-Identifier
-        </h3>
+        <h3>DID Subject</h3>
         <p>
-The <a>DID</a> for a particular <a>DID subject</a> is denoted with the
+The <a>DID</a> for a particular <a>DID subject</a> is expressed using the
 <code><a>id</a></code> property in the <a>DID document</a>.
         </p>
 
-        <p>
-<a>DID documents</a> MUST include the <code><a>id</a></code> property in the
-the topmost <a data-cite="INFRA#ordered-map">map</a> in the
-<a href="#data-model">data model</a>.
-        </p>
-
         <dl>
           <dt><dfn>id</dfn></dt>
           <dd>
 The value of <code>id</code> MUST be a <a
-data-cite="INFRA#string">string</a> that conforms to the rules in Section <a
-href="#did-syntax"></a>.
+data-cite="INFRA#string">string</a> that conforms to the rules in <a
+href="#did-syntax"></a> and MUST exist in the topmost <a
```

Comment:
Do we use `topmost` elsewhere? Is `root` better or worse?

```suggestion
href="#did-syntax"></a> and MUST exist in the root <a
```

Reply:
If I recall it got changed from root to topmost everywhere a while back, but I'm not sure why. Maybe root is not abstract enough..? But I have no problem with root.

Reply:
I'm in the process of changing everything over to `root`. I don't know if it's better or worse... just that I have rarely seen `topmost` when talking about data structures.